app/src/components/layout/Header/Header.styled.ts

```diff
@@ -12,18 +12,12 @@ export const HeaderComponentsContainer = styled.div<{
   isMobile: boolean;
 }>`
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
   width: 100%;
   max-width: ${pxToRem(1400)};
   width: ${({ isMobile }) => (isMobile ? '100%' : '98%')};
   margin: ${({ isMobile }) => (isMobile ? '0' : '0 auto')};
-
-  padding: 1.75rem 1.7rem 1.7rem 2.17rem;
-
-  @media (min-width: ${breakpoints.lg}) {
-    padding: 3.5rem 2rem 1.75rem 2rem;
-  }
+  padding-right: ${({ isMobile }) => (isMobile ? '0.8rem' : '0')};
```

Comment:
Can we use CSS to determine if we are in a mobile view here instead of passing a prop?

Reply:
Updated!